src/main/kotlin/lotto/domain/WinningResult.kt

```diff
@@ -0,0 +1,26 @@
+package lotto.domain
+
+class WinningResult(private val winningStatistics: Map<Int, Int>) {
+    fun calculateTotalPrize(): Int {
+        return winningStatistics.entries.sumOf { (matchCount, count) ->
+            calculatePrizeForMatch(matchCount, count)
+        }
+    }
+
+    private fun calculatePrizeForMatch(
+        matchCount: Int,
+        count: Int,
+    ): Int {
+        val rank = Rank.from(matchCount)
+        return rank.prize * count
+    }
+
+    // 손익률 계산
+    fun calculateProfitRate(purchaseAmount: Int): Double {
+        return calculateTotalPrize().toDouble() / purchaseAmount
+    }
+
+    fun getWinningCount(matchCount: Int): Int {
+        return winningStatistics.getOrDefault(matchCount, 0)
```

Comment:
`get operator`와 `엘비스 연산자`를 활용해서 아래오 같이 변경해볼 수 있을 것 같아요. 😃

```suggestion
        return winningStatistics[matchCount] ?: 0
```